src/app/src/test/kotlin/org/imperial/mrc/hint/integration/HintrAPIClientTests.kt

```diff
@@ -47,28 +46,21 @@ class HintrApiClientTests {
     @Test
     fun `can submit model run`() {
         val sut = HintrAPIClient(ConfiguredAppProperties(), ObjectMapper())
-        val result = sut.submit(mapOf(), mapOf())
-        assertThat(result.statusCodeValue).isEqualTo(200)
-        JSONValidator().validateSuccess(result.body!!, "ModelSubmitResponse")
+        val result = sut.submit(emptyMap(), ModelRunOptions(emptyMap(), emptyMap()))
+        assertThat(result.statusCodeValue).isEqualTo(400)
+        JSONValidator().validateError(result.body!!, "VERSION_OUT_OF_DATE")
```

Comment:
Here just confirming that we're hitting hte right endpoint. In ModelRunTests we are more complete and test the happy path.